pkg/worker/redis.go

```diff
@@ -205,7 +206,9 @@ func (w *RedisWorker) processJob(ctx context.Context, job *Job) {
 	logger.Info().Msg("Dequeued job from Redis")
 	ctx = contextLogger(ctx, job)
 	if h, ok := w.handlers[job.Type]; ok {
-		h(ctx, job)
+		metrics.ObserveBackgroundJobDuration(job.Type.String(), func() {
```

Comment:
ouch. we are using internal package function in `pkg`. Can't we do it in `job/common.go` somewhere?
Or just abandone the idea of it being under `pkg`.
Not a blocker, more of a food for thought, given we already import other internal packages :shrug:

Reply:
Yeah this is not the first time. Tho I think it is sort of okay I guess as long as you so not return an "internal" type. Unless you prove otherwise, I think this is okay.

I think somebody from the content team already asked about our Redis queue and I gave them link. Maybe this would get used by someone else?!?

We can always move this into the internal package later.